test/models/transfer_test.rb

```diff
@@ -0,0 +1,136 @@
+# == Schema Information
+#
+# Table name: transfers
+#
+#  id            :integer          not null, primary key
+#  user_id       :integer          not null
+#  department_id :integer          not null
+#  grad_date     :date             not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+require 'test_helper'
+
+class TransferTest < ActiveSupport::TestCase
+  setup do
+    @transfer = transfers(:valid)
+    file = Rails.root.join('test','fixtures','files','a_pdf.pdf')
+    @transfer.files.attach(io: File.open(file), filename: 'a_pdf.pdf')
+  end
+
+  teardown do
+    Rails.cache.clear
```

Comment:
Is the intent of this to delete the ActiveStorage file attached in `setup`? If so, I'm not sure clearing the general Rails cache will achieve that.

Reply:
Ok, let me poke on what a better way to do this would be...